src/main/java/yapchit/tasks/Task.java

```diff
@@ -0,0 +1,41 @@
+package yapchit.tasks;
+
+/**
+ * Task class representing a general task in Yapchit/
+ */
+public class Task {
+    private String name;
+    private boolean isDone;
+
+    /**
+     * Constructor of task class.
+     *
+     * @param name name of task to perform
+     */
+    public Task(String name){
+        this.name = name;
+        this.isDone = false;
+    }
+
+    /**
+     * setter function to set the tag of the class based on input.
```

Comment:
Suggestion to follow the java coding standard more closely:
```suggestion
     * Sets the tag of the class based on input.
```